R/tags.R

```diff
@@ -928,6 +997,16 @@ as.tags.shiny.tag.list <- function(x, ...) {
   x
 }
 
+#' @export
+as.tags.shiny.tag.function <- function(x, ...) {
+  as.tags(x())
+}
+
+#' @export
+as.tags.list <- function(x, ...) {
+  tagList(x)
```

Comment:
This will wrap the list into a tagList, adding one level to the tree. I think it makes more sense to just convert it to a `tagList`.

Reply:
```suggestion
  # Only non-classed lists will hit this method (classed lists will fall-through to the default method)
  do.call(tagList, x)
```